src/js/components/Select/Select.js

```diff
@@ -0,0 +1,45 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import { getClassName } from '../../utils/helpers'
+
+class Select extends PureComponent {
+  renderOptions = option => {
+    const { label, value, disabled } = option
+
+    return <option key={ value } value={ value } disabled={ disabled }>{ label }</option>
+  }
+
+  render () {
+    const { name, options, className, mods, style, inputProps } = this.props
+
+    return (
+      <label className={ getClassName(className,mods) } style={ style } htmlFor={ name }>
```

Comment:
I think this can actually be just a `<div>` wrapper instead of a `label` so that we can use the standard `FormGroup-label` as a sibling element. (I know it's written in the pattern library as a `label` but I think it will be more flexible this way - I can update the markup example)

Reply:
i can change this as well.  Does that mess up the 'htmlFor' we are using to select the inputs/selects?  We can chat more on monday as well.

Reply:
No, it shouldn't. We'd just put the `htmlFor` on the separate `-label` instead.

Reply:
Just ran a quick experiment and React won't handle the focus behavior event if you move the htmlFor to a div and not a sibling label. https://codesandbox.io/s/o9n1xm07zz

Reply:
I wasn't suggesting `htmlFor` goes on a div... but that it goes on a sibling `<label>` element rather than a wrapping `<label>` element. Are you saying React won't handle focus if the form element isn't wrapped in a `<label>`? That's crazy...

Reply:
No it just has to be a `label` in order for it to properly pass the `for` attribute. 👍